assignment/assignment.hs

```diff
@@ -31,7 +32,12 @@ factorial n = 0 -- replace me
 -- You may also need list concatenation, `++`.
 
 toDigits :: Int -> [Char]
-toDigits num = "replace me"
+
+toDigits 0 = []
+toDigits num = toDigits(truncatedResult) ++ [digit]
+  where
+    (truncatedResult, remainder) = divMod num 10
+    digit = intToDigit(remainder)
 
```

Comment:
I think I like this more than my answer:

~~~ hs
toDigits n 
    | n < 10    = [ intToDigit n ]
    | otherwise = toDigits rest ++ toDigits lowOrder 
    where
        ( rest, lowOrder ) = divMod n 10
~~~